includes/AssignPostTable.php

```diff
@@ -106,7 +108,9 @@
 
 while ($row = mysqli_fetch_row($res))
 {
-	if( isset($_POST[$row[0]]) )
+	$newRow = preg_replace('/\s+/', '_', $row[0]);
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->